comment_template.md.erb

```diff
@@ -78,6 +78,32 @@ No organizations added
 No organizations removed
 <% end %>
 
+## Memberships
+
+### Added
+<% if file.memberships_added.any? %>
+<% file.memberships_added.group_by(&:legislative_period_id).sort_by { |t, _| t }.each do |t, ms| %>
```

Comment:
I'm finding the docs for `sort` on a Hash a little confusing. I think you're almost certainly right that this will DTRT, but I think it's worth being explicit here, more for people reading the code than for `ruby` itself :)